service/ciphering/secretKeyCreator.go

```diff
@@ -0,0 +1,20 @@
+package ciphering
+
+import (
+	"crypto/sha256"
+)
+
+const keySize = 32 // length of secret key which is minimal to complete AES Encoding
+
+func PassToSecretKey(pass string) []byte {
+
+	h := sha256.New()
+	h.Write([]byte(pass))
+
+	encryptedKeyWord := h.Sum(nil)
+	if h.Size() < keySize {
+		panic("key size is incorrect ")
```

Comment:
How is that aligned with the proposition? https://github.com/retttddd/golangProjectv3/pull/1#discussion_r1133076419
Re-do this please.